.github/scripts/codegrep.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Btw, while this works for now, I really dislike shell scripts for these kinds of jobs. We should use Python (or something more readable) for this.

Reply:
Agreed. Portability is not great either, in case someone wants to run it locally. I just made sure it runs in the CI.